ste/sender-appendBlob.go

```diff
@@ -228,7 +229,7 @@ func (s *appendBlobSenderBase) GetMD5(offset, count int64) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	if response.ContentMD5 != nil && len(response.ContentMD5) > 0 {
+	if len(response.ContentMD5) > 0 {
```

Comment:
If its nil does len() method takes care of it, othewise this will result in panic

Reply:
yes it takes care of it as suggested by gosimple in the screenshot added at the top.

Reply:
Nil maps and slices return 0 when `len()`'d over.

https://go.dev/play/p/lZ55EjV-xXQ